components/measurement/DetailsBox.js

```diff
@@ -63,11 +63,11 @@ export const DetailsBox = ({ title, content, collapsed = false, ...rest }) => {
   return (
     <StyledDetailsBox width={1} {...rest} mb={3}>
       {title &&
-        <StyledDetailsBoxHeader px={3} py={2} bg='gray2' alignItems='center' onClick={onToggle}>
+        <StyledDetailsBoxHeader px={3} py={2} bg='gray2' alignItems='center' >
           <Box>
             <Heading h={4}>{title}</Heading>
           </Box>
-          <Box ml='auto'>
+          <Box ml='auto' onClick={onToggle}>
```

Comment:
Lets retain the clickable header. I am guessing you moved this to the `Box` because it toggled when you click the `Expand` button. To avoid that you can use`stopPropagation` or `preventDefault` like how it is done for the other button.